src/shared/views/map/MapRoute.js

```diff
@@ -0,0 +1,31 @@
+import React from 'react'
+
+import { MapTabs } from './index'
+
+const mapLevels = [
```

Comment:
This should really be stored in redux, with a container that passes down the map and images. Thus allowing us to bring in images from a server in the future and have different names etc.